chat.js

```diff
@@ -389,6 +397,9 @@ class CommandContext {
 		if (room.filterCaps && user.name.match(/[A-Z\s]{6,}/)) {
 			return this.errorReply(`Your username contains too many capital letters, which this room doesn't allow.`);
 		}
+        if (room.filterEmojis && emojiRegex.exec(user.name)) {
```

Comment:
Not that this matters because Unicode Technical Report 51 is more crazy than a list I found on Wikipedia - have a list of emojis: http://www.unicode.org/Public/emoji/5.0/emoji-data.txt. Yes, you read it right, ASCII `1` is emoji. I... don't even... why.